### PR TITLE
Dynamic Footer date

### PR DESCRIPTION
**Pull request by Ardent10**
##**Description**
Implemented the dynamic date for the Layer5io website. Which now does not need to be updated manually.

This PR fixes Issue: #2648 

**Notes for ### Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Abhijay007 — COMMENTED** [review]
DCO not signed

**Comment by Ardent10** [comment]
> DCO not signed

Didn't know about that, Will try to fix this.

**Comment by Ardent10** [comment]
DCO signed now.

File: src/collections/blog/2022-01-14-open-source-journey-gaurav-chadha/index.mdx [hunk 1]
```diff
@@ -19,7 +19,6 @@ import { BlogWrapper } from "../Blog.style.js";
 import Profile from "./Intern-profile.png";
 
 <BlogWrapper>
-<img src={Profile} align="right" alt="profile-img"  width="380" height="500" />
 <p>
   Hey👋, I am Gaurav Chadha, a student from New Delhi, India with lots of dreams, ambitions, and zeal to learn and grow more each and every day.
 </p>
```
Comment (Abhijay007) [inline src/collections/blog/2022-01-14-open-source-journey-gaurav-chadha/index.mdx:24]: @Ardent10 remove this file changes as it belongs to some other issue and PR is already opened for this one.

**Comment by Ardent10** [comment]
I don't understand how these other files are sneaking into this PR. I should probably make a new PR.

**Pull request closed without merging**